project-base/src/Shopsys/ShopBundle/Migrations/Version20191121171000.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\ShopBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20191121171000 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $this->sql('DELETE FROM friendly_urls WHERE slug=\'contact\'');
+        $this->sql('DELETE FROM articles WHERE name=\'Contact\' AND placement=\'footer\'');
```

Comment:
We have to be extra careful when deleting stuff. What if some meaningful content is present?

Reply:
I know, because of that I added this migration to project base, so in existing projects it should be done explicitly. I added note about it in upgrade notes.